static/js/publisher/release/helpers.js

```diff
@@ -1,15 +1,22 @@
 import { AVAILABLE } from "./constants";
+import { getChannelString } from "../../libs/channels";
 
 export function isInDevmode(revision) {
   return revision.confinement === "devmode" || revision.grade === "devel";
 }
 
 export function getChannelName(track, risk, branch) {
-  let name = risk === AVAILABLE ? risk : `${track}/${risk}`;
-  if (branch) {
-    name = `${name}/${branch}`;
+  const obj = {
+    track,
+    risk,
+    branch
+  };
+
+  if (risk === AVAILABLE) {
+    delete obj.track;
```

Comment:
Let's not overcomplicate it. The `AVAILABLE` risk/channel is artificial and we just want to return it as it is. It's never in track, never has branches.

With that we also probably don't need to make `track` optional in `getChannelString`.

```suggestion
    return AVAILABLE;
```

Reply:
Does it not need to return the branch too though - if it's present?

Reply:
The 'AVAILABLE' channel is the row where you select revisions you want to promote. It's not a real channel, it can't have branches.

<img width="1155" alt="Screenshot 2019-10-21 at 12 00 15" src="https://user-images.githubusercontent.com/83575/67196052-5da55280-f3fa-11e9-8149-f119f8bbd379.png">

Reply:
It seemed to make sense to make it a fake channel to render it similar to other channels. Now we make it more separate (put in the tabs), so probably makes sense to refactor it at some point and remove this fake channel altogether (replace with separate 'selected revisions').